tools/bash/bash_helpers.sh

```diff
@@ -5,6 +5,10 @@ function echo_and_run {
   eval "$1"
 }
 
+function run_in_dir {
+  (cd "$2" && eval "$1")
```

Comment:
commands being executed via `run_in_dir` were previously executed with `echo_and_run` so we've lost the console cmd output.